### DISCUSSION TITLE
Is Modern Flyouts still maintained?

### DISCUSSION THREAD

**Discussion by ElitePheonix2009** [ShankarBUS/ModernFlyouts#1123] [General]
Pls tell if modern flyouts is still maintained. If yes, then why are there no releases after October 20, 2021?

**Comment by Samuel12321** [comment]
hi @ElitePheonix2009 , 
ModernFlyouts IS STILL MAINTAINED.
The reason for the release delay is due to a large rebuild of the app taking place which as taken a long time.
A lot of these delays are actually thanks to microsoft, due to issues with the windows app sdk.
We hope to release 0.11 soon.

**Reply by ElitePheonix2009** [reply]
Thanks, will be looking forward for 0.11 release.

**Reply by myfix16** [reply]
Hi @Samuel12321 ,
It's glad to hear that ModernFlyouts is still maintained. Is there any estimated date for the 0.11 release? I can't wait to see it😀

**Reply by Samuel12321** [reply]
We can't wait to release it.
No eta yet

**Reply by myfix16** [reply]
Thanks, looking forward to see it😆
